### DISCUSSION TITLE
How to install .Net Maui for use with vscode on linux

### DISCUSSION THREAD

**Discussion by trampster** [dotnet/maui#7711] [Q&A]
Brandon Minnick on twitter said that you can use .NET Maui from vscode on linux. I've been trying to find instructions for how to do this, (I already have vscode and .NET 6 installed)

What/where are the instructions for doing this. How do I install .NET Maui on linux, create a project and deploy to an android device?

**Comment by trampster** [comment]
I tried:
`dotnet workload install maui`

but got :
`Workload ID maui isn't supported on this platform`

**Reply by MartyIX** [reply]
MAUI does not support linux at the moment: https://github.com/dotnet/maui/discussions/339

**Reply by trampster** [reply]
To be clear this isn't about targeting linux, this is about using linux to target Maui for android. i.e. using linux with vscode as a dev platform.

The other very useful aspect of this is being about to build Maui Android apps in linux docker images in our CI

Brandon Minnick said this was possible in this [tweet](https://twitter.com/TheCodeTraveler/status/1530245492549951489?s=20&t=tFCXfQ6GemqfHa8HDWFczw). I just want o know how to do it.

**Reply by MartyIX** [reply]
I see. In that case, I would probably try to install `dotnet workload install maui-android`. It's a wild guess though.

**Comment by Zekiah-A** [comment]
![image](https://user-images.githubusercontent.com/73035340/181598281-a8b53d8e-b89f-4c32-8cca-0cc15288b00c.png)
For me, downloading the latest prerelease .net 7 worked (arch linux binary .net download)
![image](https://user-images.githubusercontent.com/73035340/181598389-eb1cd6b7-250c-45e6-bee9-751687c9b66e.png)
https://dotnet.microsoft.com/en-us/download/dotnet/7.0
